docs/cow-protocol/concepts/batch-auctions/better-execution-prices.md

```diff
@@ -0,0 +1,7 @@
+---
+sidebar_position: 3
+---
+
+# Better Execution Prices
+
+Batch auctions are also designed to offer a better price over traditional CFMMs. Liquidity fragmentation poses a big problem for the DeFi space, forcing users to constantly check liquidity pools against each other to find the best prices. Batch auctions help solve liquidity fragmentation by matching orders peer-to-peer through Coincidence of Wants (CoWs), thus allowing them to share liquidity. If orders are not fully filled by CoWs, the protocol acts like a more traditional aggregator and finds on-chain liquidity to complete the order. In essence, batch auctions turn the entire protocol into a giant barter economy where users can either trade directly against each other.
```

Comment:
"either" without an or at the end of the sentence

Reply:
Included " In essence, batch auctions turn the entire protocol into a giant barter economy where users can either trade directly against each other or directly against the best onchain liquidity source."